autorest/preparer_test.go

```diff
@@ -70,7 +70,7 @@ func ExampleCreatePreparer() {
 	} else {
 		fmt.Println(r.URL)
 	}
-	// Output: https://microsoft.com/a/b/c
```

Comment:
@garimakhulbe Host must still be there or the URL is invalid. Opaque only means that the portion of the URI is not being further inspected (as it is scheme dependent). For the http(s) schemes, the "//host.xxx" is required.